### PR TITLE
Improve Visual Assembler and update the picture

### PR DESCRIPTION
**Pull request by officialcjunior**
The picture was a screenshot from radare2, I updated that to Rizin. 

See the output here: https://github.com/officialcjunior/book/blob/visual-mode-pic/src/visual_mode/visual_assembler.md

### CODE REVIEW

**Comment by wargio** [comment]
i do not see any file added.
be sure to do `git add path/to/file` and check if is green on `git status`

**Comment by officialcjunior** [comment]
I think I had added it fine. I had used the same file name for the new image, maybe that is what's causing this?

But, I can see the image here: https://github.com/officialcjunior/book/blob/visual-mode-pic/src/visual_mode/visual_assembler.md

File: src/visual_mode/visual_assembler.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Visual Assembler
 
-You can use Visual Mode to assemble code using `A`.
-For example let's replace the `push` by a `jmp`:
+You can use Visual Mode to assemble code (patch) using `A`.
+For example, let's `xor` the EAX register instead of EBP register, here.
+To assemble, seek to the location you want to patch and press `A`.
```
Comment (ITAYC0HEN) [inline src/visual_mode/visual_assembler.md:5]: In the next paragraph, it says: "Notice the preview of the disassembly and arrows:"

What arrows?

**Review by ITAYC0HEN — CHANGES_REQUESTED** [review]

File: src/visual_mode/visual_assembler.md (hunk 1 above)
Comment (XVilka) [reply src/visual_mode/visual_assembler.md:5]: I think it was meant to say jump/call reference lines. Which are obviously missing on the picture, probably worth to change the example to show the one with changing jumps, e.g. something like from `jz` to `jnz` or similar.

Comment (officialcjunior) [reply src/visual_mode/visual_assembler.md:5]: What about this one?
before
after

**Comment by ITAYC0HEN** [comment]
I also think that ask.bytes should be enabled to see the differences better

**Comment by officialcjunior** [comment]
> I also think that ask.bytes should be enabled to see the differences better

Alright, check it out now: https://github.com/officialcjunior/book/blob/visual-mode-pic/src/visual_mode/visual_assembler.md

**Comment by XVilka** [comment]
Not a very good example. I would see some instruction address that is already on the screen so you would see the change visually.

**Comment by officialcjunior** [comment]
You are right about that.
Changed the offset to one on the screen. 👍🏽

**Review by XVilka — APPROVED** [review]
Perfect. Looks good now 👍

**Comment by ITAYC0HEN** [comment]
Enable asm.bytes

**Review by ITAYC0HEN — CHANGES_REQUESTED** [review]
Example decision looks good, enable asm.bytes please

**Comment by officialcjunior** [comment]
> Example decision looks good, enable asm.bytes please

Ah, my bad. I can't believe I missed that.

Please check again, now.

**Review by XVilka — APPROVED** [review]

**Pull request merged**